src/main/java/com/targus/ui/controllers/InformativeController.java

```diff
@@ -0,0 +1,77 @@
+package com.targus.ui.controllers;
+
+import com.targus.base.Solution;
+import com.targus.problem.wsn.WSN;
+import com.targus.problem.wsn.WSNMinimumSensorObjective;
+import com.targus.problem.wsn.WSNOptimizationProblem;
+import com.targus.represent.BitString;
+import com.targus.ui.Mediator;
+import javafx.fxml.FXML;
+import javafx.scene.control.TextField;
+
+import java.util.HashSet;
+
+public class InformativeController {
+    @FXML
+    TextField txtSensorObjective;
+    @FXML
+    TextField txtWeightSensorObjective;
+    @FXML
+    TextField txtWeightSensorObjectiveResult;
+    @FXML
+    TextField txtConnectivityObjective;
+    @FXML
+    TextField txtWeightConnectivityObjective;
+    @FXML
+    TextField txtWeightConnectivityObjectiveResult;
+    @FXML
+    TextField txtCoverageObjective;
+    @FXML
+    TextField txtWeightCoverageObjective;
+    @FXML
+    TextField txtWeightCoverageObjectiveResult;
+    @FXML
+    TextField txtTotalResult;
+
+    private Mediator mediator;
+
+    public void setMediator(Mediator mediator) {
+        this.mediator = mediator;
+    }
+
+    public void display() {
+        WSNOptimizationProblem wsnOptimizationProblem = (WSNOptimizationProblem) mediator.getOptimizationProblem();
+        Solution solution = mediator.getSolution();
+
+        if (wsnOptimizationProblem == null || solution == null) {
+            return;
+        }
+
+        WSN wsn = (WSN) wsnOptimizationProblem.model();
+        WSNMinimumSensorObjective wsnMinimumSensorObjective = (WSNMinimumSensorObjective) wsnOptimizationProblem.getObjectiveFunction();
+        BitString bitString = (BitString) solution.getRepresentation();
+        HashSet<Integer> indexes = bitString.ones();
+
+        double sensorPenValueScaled = wsnMinimumSensorObjective.getSensorPenValueScaled(wsn, bitString.getBitSet());
```

Comment:
Well, I thought about it, but I wanted to keep InformativeController so simple that it will only contain one method so the overcrowded method. This issue is debatable, but I think the current structure is the factor that refrains InformativeController from adding to its complexity. More methods equal more complexity. Please do not hesitate to share your further thoughts.